hoot-core/src/main/cpp/hoot/core/algorithms/Sparse2dMatrix.h

```diff
@@ -64,7 +64,7 @@ template<>
   {
     size_t
     operator()(const hoot::Sparse2dCellId& k) const
-    { return cantorPairing(k); }
+    { return Tgs::cantorPairing(k); }
```

Comment:
Before I was using cantor pairing I just used xor to combine two hash values. That works well as long as the hash values are pseudo-random. Unfortunately many hash values are not. So if you xor 234 and 234 you get zero and many other similarities which in turn lead to very poor hash map performance. Cantor pairing avoids that issue.

A more thorough discussion:

http://stackoverflow.com/questions/919612/mapping-two-integers-to-one-in-a-unique-and-deterministic-way